conbench/api/benchmarks.py

```
@@ -160,7 +160,12 @@ def get(self):
     def post(self):
         """
         ---
-        description: Create a benchmark.
+        description:
+            Submit a BenchmarkResult within a specific Run (as defined by its
+            Run ID). If the Run is not known yet in the database it gets
+            implicitly created, using details provided in this request. If the
+            Run is already known then Run-specific info in this request is
+            silently ignored.
```

We might make this a bit more specific and say "If the Run ID {or whatever the key fields are that match} matches a existing run, than the rest of the fields describing a run {fields listed?} are silently ignored"